test/browser/fragments.test.js

```diff
@@ -1671,11 +1639,9 @@ describe('Fragment', () => {
 		expect(scratch.innerHTML).to.equal(htmlForFalse);
 		expectDomLogToBe(
 			[
-				'<div>.appendChild(#text)',
-				'<div>1.insertBefore(<div>3, #text)',
-				'<div>.appendChild(#text)',
-				'<div>31.insertBefore(<div>4, #text)',
```

Comment:
Most of the test changes here are just removing calls to `appendChild(#text)`. But this test case invokes something more interesting I think it worth examining.

Here, we are transitioning from a tree like:
```html
<div>1</div>
<div>2</div>
```
to
```html
<div class="wrapper">
  <div>3</div>
  <div>4</div>
</div>
```

Our diff matches `<div>1` with the new `div.wrapper`. So it sees a div with a single text child (`1`) transitioning to a div with multiple children `<div>3` and `<div>4`. So with this update, instead of creating and inserting the new children before the old children, it removes the text node and appends the new children to the now empty div.